test/dil_demo/events/pulsar_test.clj

```diff
@@ -0,0 +1,47 @@
+;;; SPDX-FileCopyrightText: 2025 Jomco B.V.
+;;; SPDX-FileCopyrightText: 2025 Topsector Logistiek
+;;; SPDX-FileContributor: Joost Diepenmaat <joost@jomco.nl>
+;;; SPDX-FileContributor: Remco van 't Veer <remco@jomco.nl>
+;;;
+;;; SPDX-License-Identifier: AGPL-3.0-or-later
+
+(ns dil-demo.events.pulsar-test
+  (:require [clojure.test :refer [deftest is]]
+            [dil-demo.events.pulsar :as sut]))
+
+(deftest wrap-auto-unsubscribe
+  (let [app   (sut/wrap-auto-unsubscribe identity :x {:site-id "test-site"})
+        store {:x {1 {:ref   "test-1"
```

Comment:
Als ik dit zo zie denk ik dat het zin heeft om een model namespace te maken waarin we state updates/queries doen, zodat je kan zien wat `{:x {1 {:ref ...}}}` betekent.